hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -325,6 +344,16 @@ public void serialize(SerializableDataOutputStream out) throws IOException {
 		out.writeByteArray(alias.toByteArray());
 		out.writeByteArray(ethereumHash);
 		writeNullableSerializable(stakingRewardsPaid, out);
+
+		if (pseudoRandomNumber >= 0) {
+			out.writeByte(PRNG_INT_OUTPUT);
+			out.writeInt(pseudoRandomNumber);
```

Comment:
We'll want to update `SeededPropertySource` to create a 0.28 version of `ExpirableTxnRecord` with both kinds of PRNG outputs.

Then these lines will be automagically covered (as long as we update [here](https://github.com/hashgraph/hedera-services/blob/b425fb5e5508a8dd0efc68b15063d176a243d8ce/hedera-node/src/test/java/com/hedera/services/state/submerkle/ExpirableTxnRecordSerdeTest.java#L63) with an appropriate case).